signing_service/signing_service/signing_service.py

```diff
@@ -102,6 +112,9 @@ def __init__(
         self.was_sigterm_caught: bool = False
         self.reconnection_counter = 0
         self.maximum_reconnection_attempts = maximum_reconnect_attempts
+        self.notifier = notifier
+        self.signing_service_daily_transaction_sum_so_far = self._get_signing_service_daily_transaction_sum_so_far()
```

Comment:
If it is in `_connect()` there is no need to call it here.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
@dybi comment:
> Also, since we want to move file reading to __init__ (separate function, i.e load_transactions_sum that is called in __init__), ...

It was `0` in `__init__` at first, but then I had to change it.

Reply:
Done, back to `0`.